src/ResourceManager/EventHub/Commands.EventHub/help/Test-AzureRmEventHubName.md

```diff
@@ -0,0 +1,139 @@
+schema: 2.0.0
+---
+
+# Test-AzureRmEventHubName
+
+## SYNOPSIS
+Checks the Availability of the given NameSpace Name or Alias (DR Configuration Name)
+
+## SYNTAX
+
+### NamespaceCheckNameAvailabilitySet (Default)
+```
+Test-AzureRmEventHubName [-Namespace] <String> [-DefaultProfile <IAzureContextContainer>]
+```
+
+### AliasCheckNameAvailabilitySet
+```
+Test-AzureRmEventHubName [-ResourceGroupName] <String> [-Namespace] <String> [-AliasName] <String>
+ [-DefaultProfile <IAzureContextContainer>]
+```
+
+## DESCRIPTION
+The **Test-AzureRmEventhubName** Cmdlet Check Availability of the NameSpace Name or Alias (DR Configuration Name)
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> Test-AzureRmServiceBusName -Namespace MyNameSapceName
+```
+
+Returns the status on availability of the namespace name 'MyNameSapceName' as True
+
+### Example 2
+```
+PS C:\> Test-AzureRmServiceBusName -Namespace MyNameSapceName
+```
+
+Returns the status on availability of the namespace name 'MyNameSapceName' as False with Reason
+
+### Example 3
+```
+PS C:\> Test-AzureRmServiceBusName -ResourceGroupName MyResourceGroup -Namespace Test123 -AliasName myAliasName
+```
+
+Returns the status on availability of the alias name 'myAliasName' under namespace 'MyNameSapceName' as True
+
+## PARAMETERS
+
+### -AliasName
+DR Configuration Name - Alias Name
+
+```yaml
+Type: String
+Parameter Sets: AliasCheckNameAvailabilitySet
+Aliases: Alias
+
+Required: True
+Position: 2
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with azure.
+
+```yaml
+Type: IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Namespace
+Eventhub Namespace Name.
+
+```yaml
+Type: String
+Parameter Sets: NamespaceCheckNameAvailabilitySet
+Aliases: NamespaceName
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+```yaml
+Type: String
+Parameter Sets: AliasCheckNameAvailabilitySet
+Aliases: NamespaceName
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ResourceGroupName
+Resource Group Name.
+
+```yaml
+Type: String
+Parameter Sets: AliasCheckNameAvailabilitySet
+Aliases: 
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+## INPUTS
+
+### System.String
+
+
+## OUTPUTS
+
+### System.Collections.Generic.List`1[[Microsoft.Azure.Commands.EventHub.Models.PSCheckNameAvailabilityResultAttributes, Microsoft.Azure.Commands.EventHub, Version=0.5.0.0, Culture=neutral, PublicKeyToken=null]]
+
+
+## NOTES
+
+## RELATED LINKS
```

Comment:
Will check over the markdown files once you have made the changes and regenerated.

Reply:
updated the markdown files, had missed a commit with big changes.